runelite-client/src/main/java/net/runelite/client/plugins/herbiboars/HerbiboarOverlay.java

```diff
@@ -145,7 +170,48 @@ public Dimension render(Graphics2D graphics)
 				}
 			}
 		}
-
+		// Draw herbiboar
+		if (plugin.isHerbiboarHarvestable())
+		{
+			Map<Integer, WorldPoint> allHerbiboarLocations = HerbiboarLocation.getAllHerbiboarLocs();
+			Integer herbiboarFinishId = plugin.getHerbiboarFinishId();
+			WorldPoint herbiboarLocation = allHerbiboarLocations.get(herbiboarFinishId);
+			LocalPoint localHerbiboarLocation = LocalPoint.fromWorld(client, herbiboarLocation);
+			final Scene scene = client.getScene();
+			final Tile[][][] tiles = scene.getTiles();
+			final Tile tile = tiles[client.getPlane()][localHerbiboarLocation.getSceneX()][localHerbiboarLocation.getSceneY()];
+			List gameObjectsOnTile = Arrays.asList(tile.getGameObjects());
+			// If we only have null objects on the tile, there is no herbiboar anymore.
+			if (Collections.frequency(gameObjectsOnTile, null) == gameObjectsOnTile.size())
+			{
+				plugin.setHerbiboarHarvestable(false);
+			}
+			for (GameObject object : tile.getGameObjects())
+			{
+				if (object != null)
+				{
+					// If we have a herbiboar. Anything between 2000 and 20000 can be assumed to be a herbiboar.
+					// Anything over 20000 is the animation during which you can walk away
+					// from the herbiboar. We don't need to highlight it then anymore.
+					if (object.getId() > 2000 && object.getId() < 20000)
```

Comment:
This code will also highlight Ammonite crabs who are on the same tile. It needs to be figured out which IDs need to be highlighted.

Reply:
Presumed fixed in 63569a8.